PyInstaller/building/makespec.py

```diff
@@ -212,6 +212,10 @@ def __add_options(parser):
     g.add_argument("--noupx", action="store_true", default=False,
                    help="Do not use UPX even if it is available "
                         "(works differently between Windows and *nix)")
+    g.add_argument("--exclude-upx", dest="exclude_upx", metavar="<FILE>", action="append", default=[],
```

Comment:
- No brackets for the meta-var. We only use it when this is not just a single word.
- No need to specify the default here. Or am I wrong for `append`?

Reply:
re-checked the docs, you're correct about the default